packages/amazonq/.vscode/launch.json

```diff
@@ -102,5 +102,23 @@
             "outFiles": ["${workspaceFolder}/dist/**/*.js", "${workspaceFolder}/../core/dist/**/*.js"],
             "preLaunchTask": "watch"
         }
+    ],
+    "inputs": [
+        {
+            "id": "disableExtensions",
+            "type": "pickString",
+            "description": "Disable extensions when running tests",
+            "options": [
+                {
+                    "value": "--disable-extensions",
+                    "label": "Yes"
+                },
+                {
+                    "value": "",
+                    "label": "No"
+                }
+            ],
+            "default": "No"
```

Comment:
Just want to verify. If I run Debug mode and the quick pick appears. If I click enter without selecting anything it will automatically choose "No"?